### PR TITLE
adding Fellow CTA

### LINKED ISSUE #152: Homepage - apply to be a fellow CTA

### PR DESCRIPTION
**Pull request by gvn**
resolves #152

### CODE REVIEW

File: app/react/pages/home/home.jsx [hunk 1]
```diff
@@ -15,6 +15,19 @@ export default React.createClass({
           item1={{text: `Maximizing access to papers, data, code, and materials so anyone can read and contribute`, icon: `/assets/img/icon-access.svg`}}
           item2={{text: `A community of researchers advocating for openness and collaboration`, icon: `/assets/img/icon-community.svg`}}
           item3={{text: `Open Science: A way for science to achieve its fullest potential`, icon: `/assets/img/icon-open.svg`}}></ThreeUp>
+
+        <div className="fellow-cta">
+          <img src="/assets/img/icon-drop.svg" className="sprinkle"/>
```
Comment (gvn) [inline app/react/pages/home/home.jsx]: The plan is to have these be parallaxed, so making them discrete elements...

**Comment by alanmoo** [comment]
The code looks ok, but it's a bit far off from the comp. Also, what issue is the jumbotron covered in? Separate issue, but the 3-up needs to move up to overlap that.

**Comment by alanmoo** [comment]
And looks really weird in the breakpoint covered by a 710px wide viewport.

**Comment by gvn** [comment]
There isn't a hompage jumbotron issue, I was just going to do smaller stuff as a PR against the main homepage ticket.

Did some tweaks...check it out.

**Comment by alanmoo** [comment]
Better. Still needs a design review, but it doesn't look broken.

**Pull request merged**